tools/Dockerfile

```
@@ -0,0 +1,34 @@
+FROM debian:stable-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+#This is required fo bypassing the tzdata installation
+ENV TZ=Europe/Bucharest
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update
+RUN apt-get install -y sudo
+RUN apt-get install -y git
+RUN apt-get install -y python
+RUN apt-get install -y curl
+RUN apt-get install -y lsb-release
+RUN apt-get install -y vim
+RUN apt-get install -y ninja-build
+RUN apt-get install -y pkg-config
+RUN apt-get install -y gcc-arm-linux-gnueabihf
+RUN apt-get install -y g++-8-arm-linux-gnueabihf
+RUN apt-get install -y ca-certificates 
+
+#Install JAVA
+RUN mkdir -p /usr/share/man/man1
```

I'd change the workdir and then do all the run commands together

---

Not sure if this is possible here, since all the following commands need to run in a different workdir

---

You could move this into the above command if you wanted to remove a layer.
```suggestion
RUN apt-get update \
	&& apt-get install -y sudo \
				git \
				python \
				curl \
				lsb-release \
				vim \
				ninja-build \
				pkg-config \
				gcc-arm-linux-gnueabihf \
				g++-8-arm-linux-gnueabihf \
				ca-certificates \
	&& apt-get clean && mkdir -p /usr/share/man/man1
```